### ISSUE TITLE
Text Clipping in Tables When Using Safari to Access the Documentation

### ISSUE THREAD

**Issue by Arafraza** [PAHFIT/pahfit#140]
Text clipping is present in many of the tables when safari is used to access the documentation.
macOS Catalina, version 10.15.7 (19H15)
Safari version 13.1.3 (15609.4.1)

<img width="719" alt="Screen Shot 2021-06-21 at 3 04 24 PM" src="https://user-images.githubusercontent.com/75290667/124011053-95c1ae80-d9ad-11eb-9afb-771713fb48b2.png">

-David Carroll

**Comment by karllark** [comment]
Yep.  This has been seen before - see issue #91.  This seems to be a problem with Safari, not sure how to fix this in the pahfit package.
